packages/connect-core/package.json

```diff
@@ -19,7 +19,8 @@
     "typescript": "^3.8.3"
   },
   "dependencies": {
-    "ethers": "^5.0.0-beta.188"
+    "ethers": "^5.0.0-beta.188",
+    "radspec-gabi": "^2.0.1"
```

Comment:
After a final round of review on the PR on Radspec we should publish the official version and update here.